ttyclock.c

```diff
@@ -61,18 +61,19 @@ init(void)
 
      /* Init default terminal color */
      if(use_default_colors() == OK)
-          ttyclock->bg = -1;
+          ttyclock.bg = -1;
 
      /* Init color pair */
-     init_pair(0, ttyclock->bg, ttyclock->bg);
-     init_pair(1, ttyclock->bg, ttyclock->option.color);
-     init_pair(2, ttyclock->option.color, ttyclock->bg);
-//     init_pair(0, ttyclock->bg, ttyclock->bg);
-//     init_pair(1, ttyclock->bg, ttyclock->option.color);
-//     init_pair(2, ttyclock->option.color, ttyclock->bg);
+     init_pair(0, ttyclock.bg, ttyclock.bg);
+     init_pair(1, ttyclock.bg, ttyclock.option.color);
+     init_pair(2, ttyclock.option.color, ttyclock.bg);
+//     init_pair(0, ttyclock.bg, ttyclock.bg);
+//     init_pair(1, ttyclock.bg, ttyclock.option.color);
+//     init_pair(2, ttyclock.option.color, ttyclock.bg);
      refresh();
 
      /* Init signal handler */
+     memset(&sig, 0, sizeof (sigaction));
```

Comment:
is this really related to the `ttyclock` object change? or should this be split out? it sure doesn't seem related to the PR's description, at least not directly: `sig` is already on the stack, FWICT.

Reply:
Done.